addons/sectorcontrol/functions/fnc_calculateDominantSide.sqf

```diff
@@ -0,0 +1,48 @@
+#include "script_component.hpp"
+/*
+    Author:
+        Bridg
+
+    Description:
+        Sorts the array of entities in area and works out who is more dominant
+
+    Parameters:
+        0: list of entities in the radius <ARRAY>
+        1: percentage for dominance <FLOAT>
+
+    Return value:
+        Nothing
+*/
+params ["_logic", "_entities", "_requiredDominance"];
+
+if !(isServer) exitWith {};
+
+// get all the men
+private _men = _entities select {_x isKindOf "Man"} apply {[_x, side _x]}; // [object, side]
+
+// work out their side
+private _menBlufor = _men select {_x#1 isEqualTo WEST};
```

Comment:
Only usage is in a `count`, so can simplify that to here
`private _bluforMenCount ={_x#1 isEqualTo WEST} count _men`